### PR TITLE
Add OSK GUI

### PR DESCRIPTION
**Pull request by al0xf**
Tested with different languages like Chinese, Japanese, Russian, and special characters like éàüåäö. Also tested cases like entering too long strings, or none at all. Finally it implements some more functionality like canceling input (just close the OSK GUI).

### CODE REVIEW

**Comment by Nekotekina** [comment]
Memleak, but not critical atm.

File: rpcs3/Gui/MsgDialog.h [hunk 1]
```diff
@@ -15,10 +15,14 @@ class MsgDialogFrame : public MsgDialogBase
 	wxStaticText* m_text;
 	wxSizer* m_sizer1;
 	wxSizer* m_buttons;
+	wxButton* osk_button_ok;
+	wxTextCtrl*  osk_text_input;
```
Comment (danilaml) [inline rpcs3/Gui/MsgDialog.h]: double space ;P
